cassovary-core/src/main/scala/com/twitter/cassovary/algorithms/similarity/Similarity.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2015 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.twitter.cassovary.algorithms.similarity
+
+import com.twitter.cassovary.graph.{GraphDir, DirectedGraph, Node, StoredGraphDir}
+import com.twitter.cassovary.graph.GraphDir.GraphDir
+import com.twitter.cassovary.util.SmallBoundedPriorityQueue
+
+case class SimilarNodes(val nodeId: Int, val score: Double) extends Ordered[SimilarNodes] {
+  import scala.math.Ordered.orderingToOrdered
```

Comment:
it imports Ordering for Tuple. 

Check out this SO Question: http://stackoverflow.com/questions/19345030/easy-idiomatic-way-to-define-ordering-for-a-simple-case-class

Is this OK? or it is inefficient?